src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @Min(1)
     private int capacity;
+    @NotNull
     private String description;
```

Comment:
checklist: Be sure you have the @Valid annotation in your controllers for the validation to take effect.

same for Movie, MovieSession and User requestDtos